### DISCUSSION TITLE
HB-RF-ETH update auf 1.3.0 machen?

### DISCUSSION THREAD

**Discussion by handy80** [OpenCCU/OpenCCU#1866] [🇩🇪 Fragen&Antworten] [answered]
Hi, 
habe heute auf meine ETH-seite geschaut und ein update wird angezeigt. Ist es sicher das Update von 1.2.9 auf 1.3.0 zu machen, wenn man mit RaspberryMatic auf aktueller Version 3.63.9.20220625 hat?
viele Grüße Holger

**Comment by jens-maus** [comment, marked as answer]
Ja, auch die Firmware Version 1.3.0 des `HB-RF-ETH` sollte problemlos mit den aktuellen RaspberryMatic Versionen funktionieren.
